curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/make-a-person.english.md

```diff
@@ -70,11 +70,13 @@ tests:
 
 ```js
 var Person = function(firstAndLast) {
+  // Only change code below this line
   // Complete the method below and implement the others similarly
   this.getFullName = function() {
     return "";
-  };
+  }; 
   return firstAndLast;
+  // Only change code above this line
```

Comment:
```suggestion
```
As discussed in the issue, there's no need to heavily restrict what the user changes.  As long as they keep

```js
var Person = function(firstAndLast) {
```

unchanged they can alter whatever they want.